vpr/src/pack/re_cluster_util.h

```diff
@@ -72,11 +72,15 @@ void remove_mol_from_cluster(const t_pack_molecule* molecule,
  * @param mode: the mode of the new cluster
  * @param clb_index: the cluster block Id of the newly created cluster block
  * @param during_packing: true if this function is called during packing, false if it is called during placement
- * @param clustering_data: A data structure containing helper data for the clustering process 
+ * @param clustering_data: A data structure containing helper data for the clustering process
  *                          (is updated if this function is called during packing, especially intra_lb_routing data member).
  * @param router_data: returns the intra logic block router data.
  * @param temp_cluster_pr: returns the partition region of the new cluster.
  * @param temp_cluster_noc_grp_id returns the NoC group ID of the new cluster
+ * @param detailed_routing_stage: options are E_DETAILED_ROUTE_FOR_EACH_ATOM (default) and E_DETAILED_ROUTE_AT_END_ONLY
+ *                                 specifies whether or not to run intra-cluster routing-based legality checking
+ *                                 after adding the molecule to the cluster; default is the more conservative option,
+ *                                 which is used in the top level re-clustering API functions
```

Comment:
Suggest changing "which is used in the top level re-clustering API functions" to something like:
"This option is passed down to lower-level routines; if it is E_DETAILED_ROUTE_AT_END_ONLY then detailed routing checks will not be done (this is useful if one detailed check can later be done on the completed cluster)."